LibraBFT/Impl/Properties/VotesOnce.agda

```diff
@@ -53,7 +54,7 @@ newVote⇒lv≡
     → Meta-Honest-PK pk → ¬ (∈BootstrapInfo-impl fakeBootstrapInfo (ver-signature sig))
     → ¬ MsgWithSig∈ pk (ver-signature sig) (msgPool pre)
     → LastVoteIs s' v
-newVote⇒lv≡ _ (step-init initSucc ini) _ send∈acts = ⊥-elim (obm-dangerous-magic' "The Contract for the init handler should say that it sends no messages, contradicting send∈acts")
+newVote⇒lv≡ _ (step-init initSucc uni) _ send∈acts = ⊥-elim (obm-dangerous-magic' "The Contract for the init handler should say that it sends no messages, contradicting send∈acts")
```

Comment:
You've pointed out that this comment is wrong.  I think you fixed it somewhere else, but please fix this one too.

Reply:
DONE. I updated this and several other places appropriately.